packages/website/content/integrations/react-hook-form.mdx

```diff
@@ -4,186 +4,41 @@ slug: '/integrations/react-hook-form/'
 section: 'integrations'
 ---
 
+import { Heading, Stack, TextLink } from '@contentful/f36-components';
+import { ExternalLinkIcon } from '@contentful/f36-icons';
+import { Props } from '@contentful/f36-docs-utils';
```

Comment:
MDX imports don't work in the new nextJS website, it won't cause the build failure, but they are redundant.

Reply:
got it, thanks